linux-gcc-5-qt-5/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM mbgl/ci:latest-linux-gcc-5
+
+# --------------------------------------------------------------------------------------------------
+# Set environment variables
+
+WORKDIR /opt
+
+RUN set -eu \
+ && sed -i 's/# deb-src/deb-src/' /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get build-dep -y qt5-default \
+ && apt-get install -y libc6-dbg libxinerama-dev libxcb-xinerama0-dev
+
+RUN set -eu \
+ && git clone git://code.qt.io/qt/qt5.git
+
+WORKDIR /opt/qt5
+
+RUN set -eu \
+ && git checkout 5.9
+
+RUN set -eu \
+ && ./init-repository --branch \
+    --module-subset=qtbase,qtxmlpatterns,qtdeclarative,qtgraphicaleffects,qtquickcontrols,qtsvg,qtimageformats,qtlocation,qtserialport,qttools
+
+WORKDIR /opt/qt5/build
+
+RUN set -eu \
+ && mkdir -p /opt/qt-5.9 \
+ && ../configure \
+    -opensource -confirm-license \
+    -nomake tests -nomake examples \
+    -debug -prefix /opt/qt-5.9 \
```

Comment:
Okay, but given this is for testing, building a debug version of Qt is fine, I assume?